rclcpp/include/rclcpp/parameter_event_handler.hpp

```diff
@@ -251,6 +251,30 @@ class ParameterEventHandler
     ParameterCallbackType callback,
     const std::string & node_name = "");
 
+  /// Configure which node parameter events will be received.
+  /**
+   * This function depends on middleware support for content filtering.
```

Comment:
i would use `rmw implementations` instead of `middleware`. but i will leave this to you.

```suggestion
   * This function depends on rmw implementation support for content filtering.
```

Reply:
Updated